src/components/NcModal/NcModal.vue

```diff
@@ -707,11 +707,14 @@ export default {
 			// wait until all children are mounted and available in the DOM before focusTrap can be added
 			await this.$nextTick()
 
-			// Init focus trap
-			this.focusTrap = createFocusTrap(contentContainer, {
+			const options = {
 				allowOutsideClick: true,
+				fallbackFocus: contentContainer,
```

Comment:
> Make sure the fallback element has a negative tabindex so it can be programmatically focused.

The mask (== content container) does not have a negative tab index.

Reply:
What if there is some focusable elements?
Will it still focus the first element or the mask then?

Reply:
Pushed